ws/websocket.go

```diff
@@ -447,7 +456,8 @@ func (server *Server) writePump(ws *WebSocket) {
 				return
 			}
 		case closed, ok := <-ws.closeSignal:
-			if !ok || closed {
```

Comment:
if you don't plan to reuse the `closeSignal` you can just close the channel which will trigger the select.

Reply:
Not being used yet by the server counterpart, since re-connection is a client feature. Will clean this up in a future PR.